modules/apps/site-initializer/site-initializer-raylife-ap/site-initializer-raylife-ap-test/src/testFunctional/tests/RaylifeAccountAvatarFunctionality.testcase

```diff
@@ -0,0 +1,133 @@
+
+		task ("Given the User go to RaylifeAp main page") {
+			RaylifeAP.goToRaylifeAPSiteWithRefresh();
+		}
+
+		task ("When the user clicks on the avatar and choose 'Dashboard' Option") {
+			RaylifeAP.userAvatarFunctionality(option = "Dashboard");
+		}
+
+		task ("Then Assert that the user is redirect to the Dashboard page") {
+			AssertElementPresent(
+				key_title = "Dashboard",
+				locator1 = "Header#H2_TITLE");
+		}
+	}
+
+	@description = " Story ID: LPS-174056 - Verify if the user can access the Account Settings page from Agent Portal by the avatar menu list."
+	@priority = 4
+	test AssertUserCanAccessAccountSettingsPage {
+		property test.name.skip.portal.instance = "RaylifeAccountAvatarFunctionality#AssertUserCanAccessAccountSettingsPage";
+
+		// Skip portal instance due to LRQA-79129. Unskip after it is resolved
+
+		task ("Given the User go to RaylifeAp main page") {
+			RaylifeAP.goToRaylifeAPSiteWithRefresh();
+		}
+
+		task ("When the user clicks on the avatar and choose 'Account Settings' Option") {
+			RaylifeAP.userAvatarFunctionality(option = "Account Settings");
+		}
+
+		task ("Then Assert that the user is redirect to the Account Settings page") {
+			AssertElementPresent(
+				key_title = "Account Settings",
+				locator1 = "Header#H2_TITLE");
+		}
+	}
+
+	@description = " Story ID: LPS-174056 - Verify if the user can access the Notification page from Agent Portal by the avatar menu list."
+	@priority = 4
+	test AssertUserCanAccessNotificationPage {
+		property test.name.skip.portal.instance = "RaylifeAccountAvatarFunctionality#AssertUserCanAccessNotificationPage";
+
+		// Skip portal instance due to LRQA-79129. Unskip after it is resolved
+
+		task ("Given the User go to RaylifeAp main page") {
+			RaylifeAP.goToRaylifeAPSiteWithRefresh();
+		}
+
+		task ("When the user clicks on the avatar and choose 'Notifications' Option") {
+			RaylifeAP.userAvatarFunctionality(option = "Notifications");
+		}
+
+		task ("Then Assert that the user is redirect to the Notifications page") {
+			AssertElementPresent(
+				key_title = "Notifications",
+				locator1 = "Header#H2_TITLE");
+		}
+	}
+
+	@description = " Story ID: LPS-174056 - Verify if the user can  Sing Out from Agent Portal by the option in avatar menu list."
+	@priority = 4
+	test AssertUserCanSingOut {
+		property test.name.skip.portal.instance = "RaylifeAccountAvatarFunctionality#AssertUserCanSingOut";
+
+		// Skip portal instance due to LRQA-79129. Unskip after it is resolved
+
+		task ("Given the User go to RaylifeAp main page") {
+			RaylifeAP.goToRaylifeAPSiteWithRefresh();
+		}
+
+		task ("When the user clicks on the avatar and choose 'Sign Out' Option") {
+			RaylifeAP.userAvatarFunctionality(option = "Sign Out");
+		}
+
+		task ("Then Assert that the user is redirect to Liferay DXP login page") {
+			AssertElementPresent(
+				key_title = "Liferay DXP",
+				locator1 = "Header#H1_TITLE");
+
+			User.firstLoginPG();
```

Comment:
Could you remove the step `User.firstLoginPG();` ? If you are already stating that the user was redirected to another page in the previous step, there is no need to log in again. One test, one intention.